opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,26 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jms</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-jms-spring</artifactId>
```

Comment:
As this just complicates things, as we then need to manually grab all the needed deps ...